Bottom: 708c1ba43e38f11313d7da3b951a94ec1bda7b11
Top:    336612f39e4c0c17e74f0e46902cb456102fc56f
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-27 22:45:34 +1300

Refresh of stream-to-extended-result

---

diff --git a/testtools/testresult/real.py b/testtools/testresult/real.py
index 5738fbf..d37cbce 100644
--- a/testtools/testresult/real.py
+++ b/testtools/testresult/real.py
@@ -11,6 +11,7 @@ __all__ = [
     'StreamResult',
     'StreamSummary',
     'StreamToDict',
+    'StreamToExtendedDecorator',
     'Tagger',
     'TestControl',
     'TestResult',
@@ -532,7 +533,8 @@ def test_dict_to_case(test_dict):
         from testtools.testcase import PlaceHolder
     outcome = _status_map[test_dict['status']]
     return PlaceHolder(test_dict['id'], outcome=outcome,
-        details=test_dict['details'], tags=test_dict['tags'])
+        details=test_dict['details'], tags=test_dict['tags'],
+        timestamps=test_dict['timestamps'])
 
 
 class StreamSummary(StreamToDict):
